CHANGELOG/CHANGELOG-1.9.md

```diff
@@ -0,0 +1,74 @@
+- Manifest files in the main branch switches to the docker.io registry for "latest" antrea images. ([#4235](https://github.com/antrea-io/antrea/pull/4235), [@antoninbas])
+- Add API support for reporting Antrea NetworkPolicy realization failure. ([#4248](https://github.com/antrea-io/antrea/pull/4248), [@wenyingd])
+- Update ResourceExport's json tag to lowerCamelCase. ([#4211](https://github.com/antrea-io/antrea/pull/4211), [@luolanzone])
+- Add clusterUUID column to S3 uploader and ClickHouseExporter to support Multi-cluster. ([#4214](https://github.com/antrea-io/antrea/pull/4214), [@heanlan])
+
+### Fixed
+
+- Fix nil pointer error when collecting support bundle from Agent fails . ([#4306](https://github.com/antrea-io/antrea/pull/4306), [@tnqn])
+- Set no-flood config for TrafficControl ports after restarting Agent to prevent packets loops. ([#4318](https://github.com/antrea-io/antrea/pull/4318), [@hongliangl])
+- Fix packet resubmission issue when AntreaProxy is enabled and AntreaPolicy is disable. ([#4261](https://github.com/antrea-io/antrea/pull/4261), [@GraysonWu])
+- Fix ownerReferences in APIExternalEntities generated from ExternalNodes. ([#4259](https://github.com/antrea-io/antrea/pull/4259), [@wenyingd])
+- Fix the issue that "MulticastGroup" API returned wrong Pods that have joined multicast groups . ([#4240](https://github.com/antrea-io/antrea/pull/4240), [@ceclinux])
+- Fix inappropriate route for IPv6 ClusterIPs in the host network when proxyAll is enabled. ([#4297](https://github.com/antrea-io/antrea/pull/4297), [@tnqn])
+- Fix log spam when there is any DNS based LoadBalancer Service. ([#4234](https://github.com/antrea-io/antrea/pull/4234), [@tnqn])
+- Remove multicast group from cache when group is uninstalled. ([#4176](https://github.com/antrea-io/antrea/pull/4176), [@wenyingd])
+- Remove redundant Openflow messages when syncing an updated group to OVS. ([#4160](https://github.com/antrea-io/antrea/pull/4160), [@hongliangl])
+- Fix nil pointer error when there is no ClusterSet found during MemberClusterAnnounce validation. ([#4154](https://github.com/antrea-io/antrea/pull/4154), [@luolanzone])
+- Fix data race when Multi-cluster controller reconciling ServiceExports concurrently. ([#4305](https://github.com/antrea-io/antrea/pull/4305), [@Dyanngg])
+- Fix memory leak in Multi-cluster resource import controllers. ([#4251](https://github.com/antrea-io/antrea/pull/4251), [@Dyanngg])
+- Fix Antrea-native policies for multicast traffic matching IGMP traffic unexpectedly. ([#4206](https://github.com/antrea-io/antrea/pull/4206), [@liu4480])
+- Fix IPsec not working in UBI based image. ([#4244](https://github.com/antrea-io/antrea/pull/4244), [@xliuxu])
+- Fix ClusterSet output when a cluster's status is empty. ([#4174](https://github.com/antrea-io/antrea/pull/4174), [@luolanzone])
```

Comment:
Fix `antctl mc get clusterset` command output when a ClusterSet's status is empty.